<launch>
  <include 
    file="$(find turtlebot_gazebo)/launch/turtlebot_world.launch"
  />
  
  <node
    pkg="naivik_robot"
    name="naivik_robot_node"
    type="naivik_robot_node"
    output="screen"
  />
</launch>